scripts/docker/gatkbase/stage_docker_base_in_cloud.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 docker_image_version"
+    exit 1
+fi
+
+IMAGE_VERSION=$1
```

Comment:
The `build_docker_base.sh` script currently has an (outdated) image version hardcoded in---maybe parametrize it there as well?

Reply:
Yes, I am definitely going to update the old build script before merging this, and also add a separate script to release a pre-staged base image to the official location